src/components/modules/Search.js

```diff
@@ -10,7 +10,7 @@ const SearchLogic = () => {
 
     const [recipes, setRecipes] = useState([]);
     const [search, setSearch] = useState('');
-    const [query, setQuery] = useState('chicken')
+    const [query, setQuery] = useState('low-carb food')
 
     useEffect(() => {
         getRecipes()
```

Comment:
Please remove the console.logs before you merge it into main 👍 
Also consider wrapping your fetch functions in a try.. catch Block to catch possible errors. This can also be done in a future sprint, just keep that in mind